### ISSUE TITLE
Popup for a GFF3Tabix feature: callbacks not working in the "Attributes" section

### ISSUE THREAD

**Issue by loraine-gueguen** [GMOD/jbrowse#1141]
Thanks for the fix of https://github.com/GMOD/jbrowse/issues/1122 in JBrowse-1.15.0

Unlike with NCList StoreClass, the callbacks ("fmtDetailValue" or "fmtDetailField") are not working in the "Attributes" section of the popup with GFF3Tabix storeClass (but they are working well in the "Primary Data" section).

Here part of the popup with a NCList StoreClass (created using flatfile-to-json.pl):

![nclist_callback](https://user-images.githubusercontent.com/16136876/43143328-d34aedb6-8f5a-11e8-87db-dab415675a6f.png)

Here the same data with the GFF3Tabix (and same track configuration -except for storeClass, urlTemplate and tbiUrlTemplate- with multiple callbacks to reformat field names, add links, remove fields):

![gfftabix_callback](https://user-images.githubusercontent.com/16136876/43143361-e4caec58-8f5a-11e8-844c-fa9b36388424.png)

**Comment by rbuels** [comment]
I think the different might be that GFF3Tabix is lowercasing all of the attribute names when it makes JBrowse features. Internally, the attribute names of JBrowse features are case insensitive when you get and set them, but maybe we can have it remember the original case for when it iterates over them.

In the meantime, if you update your `fmt*` function to use lower-case attribute names, does that fix it?

**Comment by loraine-gueguen** [comment]
Indeed, it is ok with lower-case attribute names in the fmt functions. Thanks!
I would say it is better if attributes are handled the same way in fmt functions in NClist or GFF3Tabix tracks, but there may be other constraints that prevent that.

**Comment by loraine-gueguen** [comment]
And for the GFF3Tabix, attribute handling in fmt function is not consistent between the "Primary data" section (must have same case as in the GFF file) and the "attributes" section (must have lower case).

**Comment by rbuels** [comment]
Yes, that is pretty inconsistent. We should look at cleaning that up. Categorizing as a bug.

**Comment by nathanhaigh** [comment]
Not sure if I've said this before, but...

GFF attributes are case sensitive. In fact attributes starting with an uppercase letter are reserved. The spec stats:

> Note that attribute names are case sensitive. "Parent" is not the same as "parent".
> 
> All attributes that begin with an uppercase letter are reserved for later use. Attributes that begin with a lowercase letter can be used freely by applications.
